pkg/rest/rest.go

```diff
@@ -94,18 +103,19 @@ const (
 // do performs all the network calls -
```

Comment:
If you have to write this comment then probably then the function should be renamed